docs/examples/cloudpubsubsource/README.md

```diff
@@ -29,8 +29,11 @@ events using a Push-compatible format.
    1. If you are using standard Kubernetes secrets, but want to use a
       non-default one, update `secret` with your own secret.
 
+   1. If you are [managing multiple project](../../install/managing-multiple-projects.md), be careful to specify the `project`
+   where you want your Google Cloud resources resides.
```

Comment:
```suggestion
```suggestion
    1. By default the Pub/Sub Topic will exist in the same project as your GKE cluster. However, if you are [managing multiple projects](../../install/managing-multiple-projects.md), then you can specify `spec.project`,
    which is the Google Cloud Project that the Topic exists in.
```
```